sdk/JavaScript/TestFramework/js/tests/pushTests.js

```diff
@@ -392,4 +281,27 @@ function definePushTestsNamespace() {
     };
 }
 
+function getRegistrations(channelUri, platform) {
```

Comment:
I'm not sure how I feel about these functions, aren't our functions already returning promises?  Why do we need the extra layers of wrapping?